bindings/python/pyproject.toml

```diff
@@ -13,6 +13,30 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
 
+[tool.poetry]
+name = "glaredb"
+version = "0.0.0"
+description = "glaredb bindings"
+authors = ["GlareDB <hello@glaredb.com>"]
+readme = "README.md"
```

Comment:
this shouldn't be under `tool.poetry`

If we want to keep this information, it should go under the `[project]` label. 

polars [py-project.toml](https://github.com/pola-rs/polars/blob/main/py-polars/pyproject.toml) 
datafusion-python [py-project.toml](https://github.com/apache/arrow-datafusion-python/blob/main/pyproject.toml)